src/MongoDB.Driver/Linq/Processors/SerializationBinder.cs

```diff
@@ -397,5 +410,14 @@ private Expression BindClientSideProjector(MethodCallExpression node)
 
             return node;
         }
+
+        private void SaveOutOfScopePrefix(Expression expression, string value)
```

Comment:
I took into account that this logic could be called not only for lowest expression field level  (for fields with `dot` too). In this case, it will be simply to call this method for them too.
Renaming looks good. I will do it.

Reply:
Done